CMake/resolve_dependency_modules/folly/CMakeLists.txt

```diff
@@ -17,10 +17,11 @@ cmake_minimum_required(VERSION 3.28)
 velox_set_source(FastFloat)
 velox_resolve_dependency(FastFloat CONFIG REQUIRED)
 
-set(VELOX_FOLLY_BUILD_VERSION v2025.04.28.00)
+set(VELOX_FOLLY_BUILD_VERSION v2025.12.15.00)
```

Comment:
@majetideepak @kgpai @czentgr Why do we need this change? Where can I get the hash for VELOX_FOLLY_BUILD_SHA256_CHECKSUM for v2025.06.30.00?

Reply:
You will have to download the release binary from folly releases and run sha256 checksum to get the hash. We also want the latest folly since there is an ask from folly to move away from folly::hardware_concurrency to folly::available_concurrency, so it makes sense to get it to work with latest releases.